Sources/Sentry/include/SentryViewHierarchy.h

```diff
@@ -6,9 +6,9 @@ NS_ASSUME_NONNULL_BEGIN
 
 @interface SentryViewHierarchy : NSObject
 
-- (NSArray<NSString *> *)fetchViewHierarchy;
+- (nullable NSData *)fetchViewHierarchy;
```

Comment:
I think we will need this method also in the privatedSDKOnly for RN.

Reply:
We can make it happen.